pending/lftp.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    BleachBit
+    Copyright (C) 2008-2019 Andrew Ziem
+    https://www.bleachbit.org
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+-->
+<cleaner id="lftp">
+  <label>Lftp</label>
+  <description>File transfer client</description>
+  <option id="cwd_history">
+    <label>cwd_history</label>
+    <description>Delete the working directory history</description>
+    <action command="delete" search="file" path="~/.local/share/lftp/cwd_history"/>
```

Comment:
Please use the [XDG environment variables](https://docs.bleachbit.org/cml/variables.html)